website/docs/d/network.html.markdown

```diff
@@ -43,7 +58,10 @@ The following arguments are supported:
   network objects, the ID of the distributed virtual switch for which the port
   group belongs. It is useful to differentiate port groups with same name using
   the distributed virtual switch ID.
-
+* `network_type` - (Optional) For standard virtual switch port groups, distributed virtual switch port groups and NSX port groups.
+  Networks are found by setting `network_type` to `Network`, `DistributedVirtualPortgroup`, or `OpaqueNetwork`. This is required if you have
+  multiple port groups with the same name.
+  
```

Comment:
```suggestion
* `network_type` - (Optional) The managed object type for the discovered network. This is required if you have multiple port groups with the same name. This will be one of `DistributedVirtualPortgroup` for distributed port groups, `Network` for standard (host-based) port groups, or `OpaqueNetwork` for networks managed externally, such as those managed by NSX.
```